src/coinbase/transaction.ts

```diff
@@ -88,6 +88,32 @@ export class Transaction {
     return this.model.to_address_id;
   }
 
+  /**
+   * Returns the Block Height where the Transaction is recorded.
+   *
+   * @returns The Block Height
+   */
+  blockHeight(): string | undefined {
+    return this.model.block_height;
+  }
+
+  /**
+   * Returns the Block Hash where the Transaction is recorded.
+   *
+   * @returns The Block Hash
+   */
+  blockHash(): string | undefined {
+    return this.model.block_hash;
+  }
+
+  /**
+   * Returns the transaction content of the transaction (only have ethereum transaction for now).
```

Comment:
```suggestion
   * Returns the Content of the Transaction.
```

We can enumerate language support in the developer documentation